src/TConsole.cpp

```diff
@@ -629,9 +629,9 @@ void TConsole::resizeEvent(QResizeEvent* event)
     }
 
     // don't call event in lua if size didn't change
-    bool preventLuaEvent = (mpMainDisplay->size() == mOldSize);
+    bool preventLuaEvent = (getMainWindowSize() == mOldSize);
```

Comment:
Is it appropriate to call `getMainWindowSize()` on `TConsole`s that are NOT the profile's main console? :worried: